src/cli/subcommands/snapshot_cmd.rs

```diff
@@ -35,6 +35,9 @@ pub enum SnapshotCommands {
         /// How many state-roots to include. Lower limit is 900 for `calibnet` and `mainnet`.
         #[arg(short, long)]
         depth: Option<crate::chain::ChainEpochDelta>,
+        /// Export snapshot in the experimental v2 format(FRC-0108).
+        #[arg(long)]
+        v2: bool,
```

Comment:
For the sake of future implementations and operations, I think it'd be nicer to provide an enumeration with allowed versions: `v1` and `v2`. If we add another one, we'd just need add a valueto the enum (and not handle cases like `--v2 --v3`).

Reply:
Fixed